Client/qtTeamTalk/channelstree.cpp

```diff
@@ -1498,3 +1499,22 @@ void ChannelsTree::slotUserVideoFrame(int userid, int stream_id)
     Q_UNUSED(userid);
     Q_UNUSED(stream_id);
 }
+
+bool ChannelsTree::eventFilter(QObject *object, QEvent *event)

```

Comment:
Also here use keyPressEvent() instead of eventFilter(). Installing an event-filter will slow down application performance, since all application events will go through the installed filters.

Reply:
Not succeeded for now...